static/css/style.css

```diff
@@ -33,7 +33,22 @@ footer {
 	list-style-type: none;
 	margin-right: 200px;
 	text-align: auto;
+	font-size: 18px;
 }
+.h4-footer {
+	font-size: 20px;
+	border-bottom: 1px solid #CCCCCC;
+	border-left-width: 100px;
+	margin-bottom: 0.5em;
+	padding-bottom: 0.5em;
+}
+#map-canvas {
```

Comment:
@goranche  Thanks for the warning! Like 45: Watch out for tabs - they are not alligned with others in style.css:

#map-canvas {

width: 300px;
height: 250px;
background-color: #CCC;
I think this needs to be fixed.